js/apps/admin-ui/src/user-federation/ldap/LdapSettingsGeneral.tsx

```diff
@@ -107,52 +107,38 @@ export const LdapSettingsGeneral = ({
         />
       )}
       <FormAccess role="manage-realm" isHorizontal>
-        <FormGroup
+        {/* These hidden fields are required so data object written back matches data retrieved */}
+        <KeycloakTextInput
+          hidden
+          id="kc-ui-provider-id"
+          defaultValue="ldap"
+          {...form.register("providerId")}
+        />
+        <KeycloakTextInput
+          hidden
+          id="kc-ui-provider-type"
+          defaultValue="org.keycloak.storage.UserStorageProvider"
+          {...form.register("providerType")}
+        />
+        <KeycloakTextInput
+          hidden
+          id="kc-ui-parentId"
+          defaultValue={realm}
+          {...form.register("parentId")}
+        />
+        <TextControl
+          name="name"
           label={t("uiDisplayName")}
-          labelIcon={
-            <HelpItem
-              helpText={t("uiDisplayNameHelp")}
-              fieldLabelId="uiDisplayName"
-            />
-          }
-          fieldId="kc-ui-display-name"
+          labelIcon={t("uiDisplayNameHelp")}
           isRequired
-          validated={form.formState.errors.name ? "error" : "default"}
-          helperTextInvalid={form.formState.errors.name?.message}
-        >
-          {/* These hidden fields are required so data object written back matches data retrieved */}
-          <KeycloakTextInput
-            hidden
-            id="kc-ui-provider-id"
-            defaultValue="ldap"
-            {...form.register("providerId")}
-          />
-          <KeycloakTextInput
-            hidden
-            id="kc-ui-provider-type"
-            defaultValue="org.keycloak.storage.UserStorageProvider"
-            {...form.register("providerType")}
-          />
-          <KeycloakTextInput
-            hidden
-            id="kc-ui-parentId"
-            defaultValue={realm}
-            {...form.register("parentId")}
-          />
-          <KeycloakTextInput
-            isRequired
-            id="kc-ui-display-name"
-            defaultValue="ldap"
-            data-testid="ldap-name"
-            validated={form.formState.errors.name ? "error" : "default"}
-            {...form.register("name", {
-              required: {
-                value: true,
-                message: `${t("validateName")}`,
-              },
-            })}
-          />
-        </FormGroup>
+          defaultValue="ldap"
+          rules={{
+            required: {
+              value: true,
+              message: `${t("validateName")}`,
+            },
```

Comment:
I might be wrong, but I believe this could be shortened to:
```suggestion
            required: t("validateName").toString(),
```